website/content/en/docs/building-operators/ansible/reference/scaffolding.md

```diff
@@ -5,14 +5,131 @@ weight: 20
 ---
 
 After creating a new operator project using
-`operator-sdk new --type ansible`, the project directory has numerous generated folders and files. The following table describes a basic rundown of each generated file/directory.
+` operator-sdk init --plugins ansible.sdk.operatorframework.io/v1`,
+the project directory has many generated folders and files. The following
+table describes a basic rundown of each generated file/directory.
 
+| File/Folders   | Purpose |
+| :---           | :---    |
+| Dockerfile | The Dockerfile for building the container image for the operator. |
+| Makefile | Contains make targets for building, publishing, deploying the
+container image that wraps the operator binary, and make targets for
+installing and uninstalling the CRD. |
+| PROJECT | A YAML file containing meta information for the operator. |
+| config/crd | The base CRD file and the kustomization settings. |
+| config/default | A sidecar container to be injected to the controller
```

Comment:
The way I understood it, this directory is primarily the home of `kustomization.yaml`. I don't know why the auth proxy sidecar is in here too, AFAICT, that ought to live in `config/manager/`